twine/utils.py

```diff
@@ -195,7 +195,7 @@ def get_userpass_value(cli_value, config, key, prompt_strategy=None):
     """
     if cli_value is not None:
         return cli_value
-    elif key in config:
+    elif config.get(key) is not None:
```

Comment:
Always remember you [none checks](https://res.cloudinary.com/teepublic/image/private/s--A8wli3oN--/t_Preview/b_rgb:eae0c7,c_limit,f_jpg,h_630,q_90,w_630/v1515424584/production/designs/2263782_1.jpg)